src/proto/middleware.rs

```diff
@@ -130,6 +136,19 @@ impl MiddleWare<Response> for Vec<Arc<Box<MiddleWare<Response>>>> {
 	}
 }
 
+/// Set middlewares that should be executed on a request.
+/// 
+/// # Example
+///
+/// ```
+/// fn getMainRoutes() -> Router {
+/// 	let app_middlewares = mw![middleware1, middleware2];
+///
+/// 	// Feel free to use pre-configured app middleware
+/// }
+///
+/// fn checkIf
```

Comment:
looks like you forgot to complete this? or maybe you forgot to remove?

Reply:
Sorry, I forgot to complete it. I should fix it asap.